front/lib/resources/data_source_view_resource.ts

```diff
@@ -38,13 +45,17 @@ export class DataSourceViewResource extends ResourceWithVault<DataSourceViewMode
   static model: ModelStatic<DataSourceViewModel> = DataSourceViewModel;
 
   private ds?: DataSourceResource;
+  readonly editedByUser: Attributes<User> | undefined;
```

Comment:
To align with above:
```suggestion
  readonly editedByUser?: Attributes<User>
```

Reply:
What's `Attributes<User>`?

Reply:
That's an object with the creation attributes of User - we are using this quite everywhere ? I can defintely  used `?` instead of `|undefined`